src/cmd-compress

```diff
@@ -12,10 +12,21 @@ import argparse
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import run_verbose, write_json, sha256sum_file, rm_allow_noent
 
+# Hardcoded value for compression algorithm for now
+DEFAULT_COMPRESSION_ALGO = 'xz'
```

Comment:
I'm thinking we should keep this as `gz` for now, get the patch in, and then turn it on in FCOS explicitly for now. We can then discuss separately if/when we want to switch the default in cosa itself. WDYT? (For one thing, we avoid yet another thing the RHCOS pipeline has to immediately adapt to in order to use cosa master).

Reply:
Yes, that's correct.. I should've been careful with this, will update.